src/browserlib/extract-cssdfn.mjs

```diff
@@ -88,13 +88,58 @@ const extractDlDfn = dl => {
 };
 
 
+/**
+ * Merges CSS definitions for the same property into one
+ *
+ * The function runs a few simple sanity checks on the definitions. More checks
+ * would be needed to fully validate that merging is fine.
+ *
+ * The function returns null if CSS definitions cannot be merged.
+ */
+const mergeDfns = (dfn1, dfn2) => {
+  // Definitions should be about the same property
+  if (dfn1.name !== dfn2.name) {
+    return null;
+  }
+
+  // There should never be two base definitions for the same CSS property
+  if (dfn1.value && dfn2.value) {
+    return null;
+  }
+
+  const baseDfn = dfn2.value ? dfn2 : dfn1;
+  if (!baseDfn.value && !baseDfn.newValues) {
+    return null;
+  }
+
+  const partialDfn = (baseDfn === dfn1) ? dfn2 : dfn1;
+  if (!partialDfn.newValues || partialDfn.initial) {
+    return null;
+  }
+  if (partialDfn.initial && partialDfn.initial !== baseDfn.initial) {
```

Comment:
`partialDfn.initial` can't be true-ish at this stage given the previous condition, can it?

Reply:
Ooops. Previous condition shouldn't have been there. Removed.